profiling/src/exception.rs

```diff
@@ -116,14 +148,9 @@ unsafe extern "C" fn exception_profiling_throw_exception_hook(
     // traversed. Fortunately, this behavior can be easily identified by checking for a NULL
     // pointer.
     if exception_profiling && !exception.is_null() {
-        #[cfg(php7)]
-        let exception_name = (*(*exception).value.obj).class_name();
-        #[cfg(php8)]
-        let exception_name = (*exception).class_name();
```

Comment:
I would propose using
```
        #[cfg(php7)]
        let exception = (*exception).value.obj;
```
here and completely get rid of the #[cfg] switches in track_exception

Reply:
Nice idea, I could cleanup nicely. Thanks 🙇